imagetest/test_suites/disk/block_device_naming_test.go

```diff
@@ -0,0 +1,36 @@
+//go:build cit
+// +build cit
+
+package disk
+
+import (
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+func TestBlockDeviceNaming(t *testing.T) {
+	utils.LinuxOnly(t)
+	err := exec.Command("udevadm", "trigger").Run()
+	if err != nil {
+		t.Fatal(err)
+	}
+	err = exec.Command("udevadm", "settle").Run()
+	if err != nil {
+		t.Fatal(err)
+	}
+	disks, err := os.ReadDir("/dev/disk/by-id")
+	if err != nil {
+		t.Fatal(err)
+	}
+	var disklist string
```

Comment:
what if we make disklist an actual list i.e. `var diskNames []string` and then in the final output we do `strings.Join(diskNames, " ")`?